docs/DEPLOYMENT.md

```
@@ -0,0 +1,125 @@
+*   **Ansible** (v2.16+)
+*   **QEMU / KVM**
+*   **genisoimage** (or `mkisofs`) for creating cloud-init seeds
+*   **Cloudflare Account** (for Tunnel token)
+*   **GitHub Repository Admin Access** (for Runner registration token)
+
+## 2. Building the Golden Image
+
+We use HashiCorp Packer to bake an immutable Fedora image with all necessary dependencies pre-installed.
+
+1.  Navigate to the packer directory:
+    ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)